### ISSUE TITLE
Adding fill_color to box_kwargs in iqplot.stripbox() doesn't work unless fill_alpha is also specified

### ISSUE THREAD

**Issue by kayla-jackson** [justinbois/iqplot#14]
By default, the box plots are not filled. This is accomplished by setting `fill_alpha = 0` when the box objects are created.  Specifying `fill_color` alone is not sufficient to override the default value of `fill_alpha`. The example below produces the error. 
```
import numpy as np
import pandas as pd
import iqplot

import bokeh.sampledata.autompg
import bokeh.io

# not working
p_stripbox = iqplot.stripbox(
    data = df, 
    q = "mpg", 
    cats = "origin", 
    title = "strip-box",
    box_kwargs = dict(
        fill_color = "#7C0000"
    ))

bokeh.io.show(p_stripbox)
```

The solution is to define `fill_alpha` as some non-zero value. 

```
# works fine
p_stripbox = iqplot.stripbox(
    data = df, 
    q = "mpg", 
    cats = "origin", 
    title = "strip-box",
    box_kwargs = dict(
        fill_color = "#7C0000",
        fill_alpha = 0.5
    ))

bokeh.io.show(p_stripbox)
```

iqplot.stripbox() should do this automatically. In other words, if the user passes `fill_color` to `box_kwargs`, then `fill_alpha` should not be equal to 0.

**Comment by tomroesch** [comment]
Hi Kayla, good catch. I added a fix for this to a pull request that has been waiting for @justinbois approval. I put the default `fill_alpha` to 0.5 for now of only a color is given, #13. 

Just as a hint, to make your example a working example, you need to add 
`df = bokeh.sampledata.autompg.autompg`
to your code.

**Comment by kayla-jackson** [comment]
@tomroesch 
Thank you for getting to this, and for catching the error in the example I gave! Definitely a lesson in the follies of copy-paste.

**Comment by justinbois** [comment]
Fixed with PR #13 .
